src/main/java/com/google/cloud/tools/gradle/appengine/AppEngineStandardPlugin.java

```diff
@@ -234,4 +238,23 @@ public void execute(Project project) {
       }
     });
   }
+
+  private void createShowConfigurationTask() {
```

Comment:
I mean not really, with java8 we could shorten it. But it's mostly required configuration and I prefer for it to be in the plugin configuration class.